### PR TITLE
[Update] PT-BR documentation for core-concepts to match with EN version

### PR DESCRIPTION
**Pull request by guidiego**
#### Description 
The current file is not updated with new entries in the English version. I updated the translation to match the original file.

🎃  I am making part of Hackthoberfest 🎃

> PS: The blank line is explained here: https://discord.com/channels/830184174198718474/971619665853939774/1160649751436476476

### CODE REVIEW

File: src/content/docs/pt-br/core-concepts/astro-pages.mdx [hunk 1]
```diff
@@ -25,7 +25,13 @@ Um único arquivo também pode gerar múltiplas páginas utilizando [roteamento
 
 ### Link entre páginas
 
-Escreva [elementos `<a>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/a) padrões do HTML em suas páginas Astro para fazer o link para outras páginas em seu site.
+Escreva [elementos `<a>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/a) padrões do HTML em suas páginas Astro para fazer o link para outras páginas em seu site. Utilize como seu link o **caminho URL relativo para a raiz do seu dominio**, não o caminho relativo para o arquivo.
+
+Por exemplo, para acessar `https://exemplo.com/autores/sonali/` de uma outra pagina dentrou de `exemplo.com`:
```
Comment (Yan-Thomas) [inline src/content/docs/pt-br/core-concepts/astro-pages.mdx:30]: Pequena correção da acentuação 🙌 
```suggestion
Escreva [elementos `<a>`](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/a) padrões do HTML em suas páginas Astro para fazer o link para outras páginas em seu site. Utilize como seu link o **caminho URL relativo para a raiz do seu domínio**, não o caminho relativo para o arquivo.

Por exemplo, para acessar `https://exemplo.com/autores/sonali/` de uma outra página dentro de `exemplo.com`:
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for this @guidiego, added a small suggestion!

**Comment by guidiego** [comment]
@Yan-Thomas thank you! I apply your suggestion!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @guidiego, welcome to Team Docs 🚀

**Pull request merged**